app/services/moment.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import moment from 'moment';
+
+const { computed, defineProperty } = Ember;
+
+export default Ember.Service.extend({
+  init() {
+    this._super(...arguments);
+
+    let locale = null;
+
+    defineProperty(this, 'locale', computed({
```

Comment:
Close over locale variable versus shadowing a prop.  Just preference but can be persuaded otherwise